cluster/addons/podsecuritypolicies-rbac/gce/kube-system-binding.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: ClusterRoleBinding
+metadata:
+  labels:
+    addonmanager.kubernetes.io/mode: Reconcile
+    kubernetes.io/cluster-service: "true"
+  name: kube-system:default-podsecuritypolicy:privileged
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: podsecuritypolicy:privileged
+subjects:
+- kind: ServiceAccount
+  name: default
```

Comment:
please don't give privileged access to the default service account. privileged access should require creating a service account for the purpose.